tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+skipsdist = true
+envlist = py{27,35}
+
+[tox:travis]
+2.7 = py27
+3.4 = py34
```

Comment:
We're using 3.5 in Travis, but anyway I think this section is not needed as tox-travis should already be able to map these.